easybuild/easyconfigs/t/tensorflow-probability/tensorflow-probability-0.9.0-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,49 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Authors::   Dugan Witherick (University of Warwick)
+# License::   MIT/GPL
+# $Id$
+#
+##
+
+easyblock = 'PythonBundle'
+
+name = 'tensorflow-probability'
+version = '0.9.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.tensorflow.org/probability'
+description = """TensorFlow Probability (TFP) is a library for probabilistic reasoning and statistical analysis."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+toolchainopts = {'usempi': True, 'pic': True}
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('TensorFlow', '2.1.0', versionsuffix),
+]
+
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
+    'sanity_pip_check': True,
+}
+use_pip = True
+
+exts_list = [
+    ('cloudpickle', '1.3.0', {
+     'checksums': ['38af54d0e7705d87a287bdefe1df00f936aadb1f629dca383e825cca927fa753'],
+     }),
+    (name, version, {
+     'modulename': 'tensorflow_probability',
+     'source_tmpl': 'tensorflow_probability-%(version)s-py2.py3-none-any.whl',
+     'checksums': ['66928dcbbc12987e574232a576bd9643ba7ca6b2a837f2182417286ac955d50f'],
+     'unpack_sources': False,
+     }),
+]
+
+sanity_check_commands = [
+    """python -c 'import tensorflow; import tensorflow_probability' """,
```

Comment:
this can be a one-liner, see above

Reply:
Thanks, just making and committing the changes.